docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+# Kubernetes on CenturyLink Cloud
+These scripts handle the creation, deletion and expansion of kubernetes clusters on CenturyLink Cloud.
+
+You can accomplish all these tasks with a simple single command. And, for those interested in what's under the covers, we used Ansible to perform these tasks and we have made these Ansible playbooks available as well.
```

Comment:
remove "simple"

Reduce to just: "We have made the Ansible playbooks used to perform these tasks available in [the Ansible docs](LINKURLHERE)."